commands-yml/commands/element/attributes/attribute.yml

```diff
@@ -0,0 +1,79 @@
+---
+name: Get Element Attribute
+short_description: Get the value of an element's attribute
```

Comment:
Do we have any information about what attributes are available?

Reply:
I'm not sure. I couldn't find a document that listed all of that.